### ISSUE TITLE
doc missing: initFirebase call needed before anything

### ISSUE THREAD

**Issue by loicginoux** [arnesson/cordova-plugin-firebase#827]
Hello, 
First, thanks for your plugin. 
I think it would be helpful to add to the README doc that a call to initFirebase() is necessary before anything. I had issues at initialisation and could not find anywhere the cause of the problem.

If it can help others  here is a "get started" code:

```javascript
      // init firebase
      window.FirebasePlugin.initFirebase()
      // grant permission 
      window.FirebasePlugin.hasPermission(function(data){
        if (!data.isEnabled) {
          window.FirebasePlugin.grantPermission();
        };
      });
      // get FCM registration token
      window.FirebasePlugin.getToken(function(token) {
          // save this server-side and use it to push notifications to this device
          console.log(`FirebasePlugin.getToken: ${token}`);
      }, function(error) {
          console.log(`FirebasePlugin.getToken error: ${error}`);
      });

      // do something when user open notification
      window.FirebasePlugin.onNotificationOpen(function(notification) {
          console.log(`FirebasePlugin.onNotificationOpen: ${notification}`);
      }, function(error) {
          console.log(`FirebasePlugin.onNotificationOpen error: ${error}`);
      });
```

NOTE: for meteor, it needs to be wrapped in a Meteor.startup function to work too.

**Comment by soumak77** [comment]
@loicginoux would you mind submitting a PR with updates to the README?

**Comment by briantq** [comment]
I think an alternative fix here would be to initialize Firebase by default, at least Firebase proper.  This was disabled in the Crashlytics PR submitted a few days ago.  I understand that for GDPR compliance one might not want to enable crash reporting and making that configurable makes sense.  

For things that require a call into the plugin to get data (with Messaging you need to get the Token), it seems like initializing it when the call is made would make sense.  That way you only have to explicitly start the components that you never interact with, but everything else works seamlessly (and doesn't required a documentation update)

**Comment by soumak77** [comment]
This is being fixed via #830.  If we can have some people help test out that PR, we can get the fix published sooner.

**Comment by soumak77** [comment]
This should be fix via `v1.1.4`.  Please be sure to read my [comment](https://github.com/arnesson/cordova-plugin-firebase/issues/822#issuecomment-419480916) above needing to revert some changes to make it work.
